golem-worker-executor-base/src/durable_host/mod.rs

```diff
@@ -573,6 +585,7 @@ impl<Ctx: WorkerCtx + DurableWorkerCtxView<Ctx>> DurableWorkerCtx<Ctx> {
                                         .on_worker_update_succeeded(
                                             target_version,
                                             component_metadata.size,
+                                            HashSet::new(), // TODO: get it from component_metadata
```

Comment:
Part of this pr or as a followup?

Reply:
No, I'm fixing these - just marked as "ready" too soon